frontend/animal/VocabBrowser/App.js

```diff
@@ -3,7 +3,8 @@ import {inject, observer} from "mobx-react";
 import PropTypes from "prop-types";
 import React, {Component} from "react";
 
-import Table from "./Table";
+import EhvTable from "./EhvBrowser/Table";
```

Comment:
As of now, the term tables are very similar. I went off of the assumption that tables may vary more widely in the future, and didn't want any logic to get muddy, so I'm passing in the full table instead of creating smaller custom components. Again, open to feedback!